npm-shrinkwrap.json

```diff
@@ -351,9 +351,9 @@
       "resolved": "https://registry.npmjs.org/bluebird/-/bluebird-3.4.1.tgz"
     },
     "bn.js": {
-      "version": "4.11.6",
+      "version": "4.11.5",
```

Comment:
Oh, I might have. There were conflicts, so I regenerated the file. I don't know why that wouldn't keep the dependency tree...